README.md

```diff
@@ -8,7 +8,7 @@ crates.
 
 Here is the list of device backends that we support:
 
-- [I2C](https://github.com/rust-vmm/vhost-device/blob/master/src/i2c/README.md)
+- [I2C](https://github.com/rust-vmm/vhost-device/blob/master/i2c/README.md)
```

Comment:
nit: master should also be replaced with main.

Reply:
Will fix that, thanks for noticing it. Though the link still works somehow, I think github is making it work somehow or something else.

Reply:
Yup, no rush. This seems to be the case: https://github.blog/changelog/2020-07-17-links-to-deleted-branches-now-redirect-to-the-default-branch/.